src/main/java/ladder/domain/Participants.java

```diff
@@ -28,6 +29,15 @@ private static void validateParticipantNames(String participantNames){
         }
     }
 
+    public Participant getParticipantByName(String name){
+        if(name.equals(RESULT_ALL)){
```

Comment:
```suggestion
        if(RESULT_ALL.equals(name)){
```

로 변경해보면 어떨까요? 

참고한 게시글 입니다 ㅎㅎ
https://www.javacodegeeks.com/2012/06/avoid-null-pointer-exception-in-java.html

Reply:
공유 감사합니다! 변경해야겠네요